### PR TITLE
Feature rwop accessmode support for csi-powerscale

### PR DESCRIPTION
**Pull request by bpjain2004**
# Description
Documentation update about new feature support in csi-powerscale - RWOP access mode

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| dell/csm#80|

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Review by randeepsharma — DISMISSED** [review]
LGTM

**Review by nitesh3108 — DISMISSED** [review]
LGTM

**Review by prablr79 — APPROVED** [review]

**Comment by prablr79** [comment]
@shanmydell can you please merge this PR ?

File: content/docs/csidriver/features/powerscale.md [hunk 1]
```diff
@@ -527,3 +527,28 @@ In the case of operator-based installation, default permission for powerscale di
 
 Other ways of configuring powerscale volume permissions remain the same as helm-based installation.
 
+## Single Pod Access Mode for PersistentVolumes- ReadWriteOncePod (ALPHA FEATURE)
+
+Use `ReadWriteOncePod(RWOP)` access mode if you want to ensure that only one pod across whole cluster can read that PVC or write to it. This is only supported for CSI Driver for PowerScale 2.1 and Kubernetes version 1.22+.
```
Comment (randeepsharma) [inline content/docs/csidriver/features/powerscale.md:532]: could write Dell EMC PowerScale 2.1.0 instead of PowerScale 2.1

Comment (shanmydell) [reply content/docs/csidriver/features/powerscale.md]: addressed

**Review by shanmydell — APPROVED** [review]

File: content/docs/csidriver/features/powerscale.md (hunk 1 above)
Comment (randeepsharma) [reply content/docs/csidriver/features/powerscale.md]: This isnt addressed, probably u meant to comment else where.

**Review by prablr79 — APPROVED** [review]

File: content/docs/csidriver/features/powerscale.md (hunk 1 above)
Comment (shanmydell) [reply content/docs/csidriver/features/powerscale.md]: @randeepsharma https://github.com/dell/csm-docs/pull/90/commits/4faac67cf8ffad06224681cd744c3df73130acfa. Please refresh the link, it is addressed part of the above commit.

**Pull request merged**